src/Service/Nph/NphOrderService.php

```diff
@@ -372,4 +378,65 @@ public function getParticipantOrderSummaryByModuleAndVisit(string $participantid
         }
         return $returnArray;
     }
+
+    public function saveOrderFinalization($formData, $sample)
```

Comment:
I noticed that most of the parameters in this service are not type-hinted.  Can you add type-hints for these parameters?

Reply:
Done!